app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        try:
+            if "vaccine" not in visitor:
+                raise NotVaccinatedError("NotVaccinatedError")
```

Comment:
The error message 'NotVaccinatedError' is not informative. Consider providing a more descriptive message that explains why the error is being raised.